open-metadata-resources/open-metadata-deployment/charts/vdc/templates/apache-atlas.yaml

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project.
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ include "myapp.name" . }}
+    helm.sh/chart: {{ include "myapp.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: atlas-service
+  name: {{ include "myapp.fullname" . }}-atlas-service
+
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: 21000
+      targetPort: 21000
+      nodePort: {{ .Values.service.nodeport.atlas }}
+  selector:
+    app.kubernetes.io/name: {{ include "myapp.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: atlas-service
+...
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  name: {{ include "myapp.fullname" . }}-atlas
+  labels:
+    app.kubernetes.io/name: {{ include "myapp.name" . }}
+    helm.sh/chart: {{ include "myapp.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: atlas-service
+
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "myapp.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: atlas-service
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app.kubernetes.io/name: {{ include "myapp.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: atlas-service
+    spec:
+      containers:
+        - name: apache-atlas
+          image: "{{ .Values.image.atlas.repository }}:{{ .Values.image.atlas.tag }}"
+          imagePullPolicy: {{ .Values.image.atlas.pullPolicy }}
+          ports:
+            - containerPort: 21000
+          livenessProbe:
+            httpGet:
+              path: /login.jsp
+              port: 21000
+            initialDelaySeconds: 120
```

Comment:
Yes, I found the timeouts inconsistent, but also that it was better to let the pod restart itself due to failing the probes rather than waiting for very long times (even after which certain containers, like Atlas, didn't start due to what appeared to be some internal race issue).

From reading about the probes, this sounded like precisely what they're there for =)